src/interfaces/cli.js

```diff
@@ -15,8 +15,7 @@ class CliClient {
   }
 
   sendMessage(channel, message) {
-    console.log(`[${this.nick} => ${channel}]\n${message}`);
-    console.log(`[${channel} => ${this.nick}]`);
+    console.log(message);
```

Comment:
No, I mean this method will not be called instantly like it should have in
the case of a static behavior. That might give the user an impression of a
blocking call. Therefore, I decided to drop this altogether.